mono/metadata/sgen-tarjan-bridge.c

```diff
@@ -94,13 +92,21 @@ enum {
 /*
 Optimizations:
 	We can split this data structure in two, those with bridges and those without
+	(and only bridgeless need to record incoming_colors)
 */
+#define API_INDEX_BITS        26
+#define INCOMING_COLORS_BITS  5
+#define API_INDEX_MAX         ((1<<API_INDEX_BITS)-1)
```

Comment:
Does mono have a "static assert" macro? If so, we should assert that `(API_INDEX_BITS + INCOMING_COLORS_BITS) <= 31`.